run-ib-pr-matrix.sh

```diff
@@ -78,6 +78,17 @@ pushd "$WORKSPACE/matrix-results"
     fi
   done
   set -x
+
+  LOG=runall-report-step123-.${BUILD_ID}.log
```

Comment:
@gartung , I would suggest to run tis only if `$CMD_OPTS` container `maxmem_profile` option e.g
```
if [[ $CMD_OPTS =~ maxmem_profile ]] ; then
  #do the processing....
fi
```

Reply:
OK